packages/training/python/config/reward_weights.yaml

```
@@ -0,0 +1,100 @@
+
+# Temporal credit decay
+temporal:
+  decay_rate: 0.9   # Credit weight decays by 10% per step
+  min_weight: 0.1   # Don't credit decisions below this weight
+
+# Volatility dampening
+volatility:
+  low: 2.0          # Standard deviation threshold for low volatility
+  high: 15.0        # Standard deviation threshold for high volatility
+  dampening_factor: 0.5  # Max dampening at high volatility (50% signal reduction)
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)